contracts/Bridge.sol

```diff
@@ -103,6 +105,7 @@ contract Bridge is DelphinusBridge {
     function addVerifier(address vaddr) public returns (uint256) {
         ensure_admin();
         uint256 cursor = verifiers.length;
+        verifierID = verifiers.length;
```

Comment:
This piece of codes are not clear for human reading.
verifierID is set for the 'before add' length, which in fact it is used for currently active verifier index.
And also need think about whether it should put after the require(verifiers.length < 255) check. (I think yes)
So for me, it should be like:
require(verifiers.length < 255, "Verifier index out of bound");
verifiers.push(DelphinusVerifier(vaddr));
verifierID = verifiers.length - 1;


Also need check whether verifiers.length < 255, this 255 is because any reason, if no reason, maybe change it to larger one (like 16,384 will be better)

Reply:
After discussion, we have something more need to do:
1. change vid from unit8 to unit16 so we have more chances to upgrading verifier (255 is too small)
2. We have security issue here as we do not force verify function to use latest verifier and it will cause security issue. (Hacker can attach it by specifiy old vid and use old buggy verifier)
We will create new item for those things and commit this PR.